app/src/main/java/com/mapbox/dash/example/ui/SampleEtaView.kt

```diff
@@ -178,9 +178,9 @@ private fun SampleTripProgress(
                     .align(BiasAlignment(horizontalBias = waypointData.iconBias, verticalBias = 0.0f)),
                 painter = painterResource(
                     if (waypointData.isChargingStation) {
-                        R.drawable.ic_navux_trip_progress_charging
+                        com.mapbox.dash.example.R.drawable.ic_navux_trip_progress_charging
```

Comment:
Not a good idea to copy these icons, since they use style attributes. I recommend doing the same as our internal example project: use icons from material library. `getStateOfChargeIcon` can be removed since it is not important

Reply:
Ok, I've reverted all icons except `ic_battery` since it's still used in our internal showcase app as well. Let's remove it in the next version.